@AH-64D Apache Longbow/Addons/fza_ah64_sfmplus/functions/simpleRotor/fn_simpleRotor.sqf

```diff
@@ -0,0 +1,172 @@
+/* ----------------------------------------------------------------------------
+Function: fza_sfmplus_fnc_simpleRotor
+
+Description:
+	Simple rotor provides a simple, grounded in reality simulation of a
+    helicopters rotor. Translational Lift, Ground Effect and Vortex Ring State
+    are all simulated.
+
+Parameters:
+	_heli - The helicopter to get information from [Unit].
+
+Returns:
+	...
+
+Examples:
+	...
+
+Author:
+	BradMick
+---------------------------------------------------------------------------- */
+params ["_heli", "_deltaTime", "_altitude", "_temperature", "_dryAirDensity"];
+
+private _colorRed = [1,0,0,1]; private _colorGreen = [0,1,0,1]; private _colorBlue = [0,0,1,1]; private _colorWhite = [1,1,1,1];
+
+DRAW_LINE = {
+    params ["_heli", "_p1", "_p2", "_col"];
+    drawLine3D [_heli modelToWorldVisual _p1, _heli modelToWorldVisual _p2, _col];
+};
+
+private _rtrPos              = [0.0, 2.06, 0.70];
+private _rtrHeightAGL        = 3.606;   //m
+private _rtrDesignRPM        = 289.0;
+private _rtrRPMTrimVal       = 1.01;
+private _rtrGearRatio        = 72.29;
+private _rtrNumBlades        = 4;
+
+private _bladeRadius         = 7.315;   //m
+private _bladeChord          = 0.533;   //m
+private _bladePitch_min      = 1.0;     //deg
+private _bladePitch_max      = 19.0;    //deg
+
+private _rtrPowerScalarTable = [
+                                [   0, 1.777]
+                               ,[2000, 1.617]
+                               ,[4000, 1.530]
+                               ,[6000, 1.377]
+                               ,[8000, 1.284]
+                               ];
+private _rtrGndEffModifier   = 0.238;
+private _rtrThrustScalar_min = 0.120;
+private _rtrThrustScalar_max = 1.830;   //20,200lbs @ 6700ft, 15 deg C and 0.9 collective
+
+private _altitude_max        = 30000;   //ft
+private _baseThrust          = 102302;  //N - max gross weight (kg) * gravity (9.806 m/s)
+
+//Thrust produced 
+private _bladePitch_cur                = _bladePitch_min + (_bladePitch_max - _bladePitch_min) * fza_sfmplus_collectiveOutput;
+private _bladePitchInducedThrustScalar = _rtrThrustScalar_min + ((1 - _rtrThrustScalar_min) / _bladePitch_max)  * _bladePitch_cur;
+(_heli getVariable "fza_sfmplus_engPctNP")
+    params ["_eng1PctNP", "_eng2PctNp"];
+private _inputRPM                  = _eng1PctNP max _eng2PctNp;
```

Comment:
No, the the power turbine drives the rotor, so the maximum of the two engines Np is what keeps the rotor where it's supposed to be. That's why the input RPM for the rotor is the highest of the two engines. There are cases where if a single engine is retarded to idle, then that engine would be at idle Np of 57% on the ground and 95% in the air. When the proper engine simulation is implemented, this will actually come from the transmission.